src/components/CircularOutline/CircularOutline.js

```diff
@@ -144,10 +149,10 @@ class RoundedOutline extends Component<Props, State> {
                   innerRef={node => (this.shapeNode = node)}
                   x={0}
                   y={0}
-                  width={width}
-                  height={height}
+                  width="100%"
+                  height="100%"
```

Comment:
Is this change necessary? I believe we want to use the width/height provided

Reply:
This is the core "fix" that allows the component to respond to text size changes. I mean, with the current method, it won't need the "width" prop. Maybe we should get rid of it. 

We do still need the height prop, of course. But the width is now not necessary, as the Chrome renderer automatically calculates it

Reply:
Ohh I see; I think we can solve for this the same way we did with `pathLength` :) Adding comments above